utilities/aap.py

```
@@ -0,0 +1,619 @@
+    return result.stdout.strip()
+
+
+def is_awx_installed() -> bool:
+    """Check if AWX Operator is already installed via Helm.
+
+    Returns:
+        bool: True if the AWX Helm release exists in the awx namespace.
+    """
+    try:
+        output = _run_shell(f"helm list -n {AWX_NAMESPACE} -q")
+        return AWX_HELM_RELEASE_NAME in output
```

Ruff (0.15.20)

[warning] Consider moving this statement to an `else` block

(TRY300)